tests/test_model.py

```
@@ -0,0 +1,37 @@
+import asyncio
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)